macos/Sources/App/macOS/AppDelegate.swift

```diff
@@ -199,6 +199,13 @@ class AppDelegate: NSObject,
         ]
 
         let center = UNUserNotificationCenter.current()
+        // Request notification authorization, including badge permission
+        center.requestAuthorization(options: [.alert, .sound, .badge]) { (granted, error) in
```

Comment:
Does this trigger a consent notification? If so, we should delay this until the first bell, I think.